modules/camera/camera_picamera2.py

```diff
@@ -4,6 +4,8 @@
 
 import numpy as np
 
+from . import camera_configurations
+
```

Comment:
Can you move these to line 15 (so that our libraries are grouped together, and 2 lines before constants)